docs/lib/push-notifications/fragments/js/getting-started.md

```diff
@@ -166,25 +166,21 @@ Push Notifications category is integrated with [AWS Amplify Analytics category](
 ## Setup for iOS
 
 1. For setting up iOS push notifications, you need to download and install Xcode from [Apple Developer Center](https://developer.apple.com/xcode/).
+
 2. Setup iOS Push Notifications and create a p12 certificate as instructed here in [Amazon Pinpoint Developer Guide](https://docs.aws.amazon.com/pinpoint/latest/developerguide/apns-setup.html).
- 
+
 3. Create a native link on a React Native app:
 
-    ```javascript
-    $ react-native init myapp
-    $ cd myapp
-    $ npm install
-    $ npm install aws-amplify && npm install @aws-amplify/pushnotification
-    $ react-native link @aws-amplify/pushnotification
-    $ react-native link amazon-cognito-identity-js # link it if you need to Sign in into Cognito user pool
+    ```bash
+    react-native init myapp
+    cd myapp
+    npm install
+    npm install aws-amplify
+    npm install @aws-amplify/pushnotification
+    npm install @react-native-community/push-notification-ios
```

Comment:
Can we change this to a single command like:
```
npm install aws-amplify @aws-amplify/pushnotification @react-native-community/push-notification-ios
```

Reply:
We had it like this in the first pr - jpignata was keen for readability so we line seperated the entries

Reply:
Thanks Matt. What Ashish is asking for is one invocation of `npm`. The issue I had was a long command of multiple `npm` invocations separated with `&&`. Ashish's suggestion LGTM, though I might ensure a user can read it without scrolling, to wit:

```bash
npm install aws-amplify \
            @aws-amplify/pushnotification \
            @react-native-community/push-notification-ios
```

Reply:
Have just submitted the change on formatting